### PR TITLE
Add release notes for Bloop 1.0.0-M7

### PR DESCRIPTION
**Pull request by Duhemm**

### CODE REVIEW

File: notes/v1.0.0-M7.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Bloop v1.0.0-M7
+
+This is the seventh milestone of Bloop. This release focuses on improving the user experience and
+fixes several important bugs.
+
+## Release highlights
+
+#### Improvements to our installation process
```
Comment (olafurpg) [inline notes/v1.0.0-M7.md]: Simpler installation

File: notes/v1.0.0-M7.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+[@valencik][@valencik] contributed a fix to our installation script, in order to make it compatible with both
+Python 2 and 3. Thanks to his changes, the installation script now runs smoothly on all
+configurations. The work happened in [#343][#343]. Thanks [@valencik][@valencik] :tada: !
+
+#### Fixes and improvements to sbt-bloop
```
Comment (olafurpg) [inline notes/v1.0.0-M7.md]: Improved sbt-bloop

File: notes/v1.0.0-M7.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+sbt-bloop used to define a task called `installBloop`, which has been renamed to `bloopInstall`. The
+reason for the renaming is to improve the discoverability of this task. For more details, see the
+discussion in [#341][#341] and [#346][#346].
+
+#### Fixes and improvements to file watching
```
Comment (olafurpg) [inline notes/v1.0.0-M7.md]: Better file watching

File: notes/v1.0.0-M7.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+#### Fixes and improvements to the task engine
+
+After a compilation failure, for instance, Bloop would still try (and fail) to run a project. This
+problem has been fixed by improvements to how different tasks are combined within Bloop. See [#347][#347]
+for all the details.
+
+#### Autocompletion for Bash and zsh
+
+In order to make Bloop more enjoyable to use, we added command line completions for Bloop,
```
Comment (olafurpg) [inline notes/v1.0.0-M7.md]: gif or it didn't happen 😄

File: notes/v1.0.0-M7.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+Support for listener has also been added, so that bloop knows when a client has disconnected and can
+stop watching files. For more details, see [#345][#345].
+
+#### Fixes and improvements to the task engine
+
+After a compilation failure, for instance, Bloop would still try (and fail) to run a project. This
+problem has been fixed by improvements to how different tasks are combined within Bloop. See [#347][#347]
+for all the details.
+
+#### Autocompletion for Bash and zsh
```
Comment (olafurpg) [inline notes/v1.0.0-M7.md]: Move this to the top, it's awesome!

File: notes/v1.0.0-M7.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+platforms, our [installation instructions][installation instructions] explain how to install the completion.
+
+For details about the work, see [#353][#353].
+
+## Install Bloop
+
+To install the latest version of Bloop, you can either use Homebrew:
+
+```sh
+$ brew install bloop
```
Comment (olafurpg) [inline notes/v1.0.0-M7.md]: no custom tap needed?

File: notes/v1.0.0-M7.md (hunk 4 above)
Comment (olafurpg) [reply notes/v1.0.0-M7.md]: I personally use https://gifox.io/

Is it possible to turn asciinema to gifs?

**Comment by Duhemm** [comment]
Thanks for your feedback @olafurpg !

@jvican Do you want to add something?

**Pull request merged**